### ISSUE TITLE
smallrye-graphql-cdi triggers "Detected bean archive with bean discovery mode of 'all'" warning

### ISSUE THREAD

**Issue by rsvoboda** [quarkusio/quarkus#33825]
### Describe the bug

smallrye-graphql-cdi triggers "Detected bean archive with bean discovery mode of 'all'" warning

`2023-06-05 11:36:30,243 WARN  [io.qua.arc.dep.BeanArchiveProcessor] (build-50) Detected bean archive with bean discovery mode of 'all', this is not portable in CDI Lite and is treated as 'annotated' in Quarkus! Path to beans.xml: io.smallrye:smallrye-graphql-cdi:/META-INF/beans.xml`

@manovotn changed the discovery mode for many of the beans.xml in graphql repo in https://github.com/smallrye/smallrye-graphql/commit/37eee6388e1b27b47ffb37e1f9e1cc1300caa9d8

https://github.com/smallrye/smallrye-graphql/blob/main/server/implementation-cdi/src/main/resources/META-INF/beans.xml still uses `bean-discovery-mode="all"`

Also there are some tests using that too:
```
server/implementation-cdi/src/main/resources/META-INF/beans.xml:       version="4.0" bean-discovery-mode="all">

server/implementation-cdi/src/test/resources/beans.xml:       version="4.0" bean-discovery-mode="all">
server/integration-tests-jdk16/src/test/resources/META-INF/beans.xml:       version="4.0" bean-discovery-mode="all">
server/integration-tests/src/test/resources/META-INF/beans.xml:       version="4.0" bean-discovery-mode="all">
```

### Expected behavior

No "Detected bean archive with bean discovery mode of 'all'" warning

### Actual behavior

"Detected bean archive with bean discovery mode of 'all'" warning is present

### How to Reproduce?

 - Pull https://github.com/quarkus-qe/quarkus-startstop
 - Run `mvn clean verify -Ptestsuite-no-native -Dtest=CodeQuarkusTest#supportedExtensionsSubsetA`

### Output of `uname -a` or `ver`

macOS

### Output of `java -version`

Java 17

### Quarkus version or git rev

3.1.0.Final

**Comment by jmartisk** [comment]
Oh man, you're right, I changed it by mistake, sorry.
I'll re-fix this asap.

**Comment by gsmet** [comment]
@jmartisk let's make sure we update to a fixed version before Quarkus 3.2 release.
